src/pages/[platform]/build-a-backend/auth/use-existing-resources/index.mdx

```diff
@@ -0,0 +1,194 @@
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from "@aws-amplify/backend"
+import { UserPool, UserPoolClient } from "aws-cdk-lib/aws-cognito"
+
+/**
+ * @see https://docs.amplify.aws/react/build-a-backend/ to add storage, functions, and more
+ */
+const backend = defineBackend({})
+
+const authStack = backend.createStack("ExistingAuth")
+const userPool = UserPool.fromUserPoolId(
+  authStack,
+  "UserPool",
+  "<your-cognito-user-pool-id>"
+)
+const userPoolClient = UserPoolClient.fromUserPoolClientId(
+  authStack,
+  "UserPoolClient",
+  "<your-cognito-user-pool-client-id>"
+)
+// Cognito Identity Pools can be referenced directly
+const identityPoolId = "<your-cognito-identity-pool-id>"
+
+backend.addOutput({
+  auth: {
+    aws_region: authStack.region,
+    user_pool_id: userPool.userPoolId,
+    user_pool_client_id: userPoolClient.userPoolClientId,
+    identity_pool_id: identityPoolId,
+    // this property configures the Authenticator's sign-up/sign-in views
+    username_attributes: ["email"],
+    // this property configures the Authenticator's sign-up/sign-in views
+    standard_required_attributes: ["email"],
+    // this property configures the Authenticator confirmation views
+    user_verification_types: ["email"],
+    unauthenticated_identities_enabled: true,
+    // this property configures the validation for the Authenticator's password field
+    password_policy: {
+      min_length: 8,
+      require_lowercase: true,
+      require_uppercase: true,
+      require_numbers: true,
+      require_symbols: true,
+    },
+  },
+})
+```
+
+{/* double-filter as a placeholder pending examples for other platforms */}
+{/* when others are added, remove the outer filter to keep the same prose */}
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
+
+## Configure the client library directly
+
+Alternatively, you can use existing resources without an Amplify backend.
+
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
+
+```ts title="src/main.ts"
+import { Amplify } from "aws-amplify"
+
+Amplify.configure({
+  Auth: {
+    Cognito: {
+      userPoolId: "<your-cognito-user-pool-id>",
+      userPoolClientId: "<your-cognito-user-pool-client-id>",
+      identityPoolId: "<your-cognito-identity-pool-id>",
+      loginWith: {
+        email: true,
+      },
+      signUpVerificationMethod: "code",
+      userAttributes: {
+        email: {
+          required: true,
+        },
+      },
+      allowGuestAccess: true,
+      passwordFormat: {
+        minLength: 8,
+        requireLowercase: true,
+        requireUppercase: true,
+        requireNumbers: true,
+        requireSpecialCharacters: true,
+      },
+    },
+  },
+})
+```
+
+</InlineFilter>
+<InlineFilter filters={["flutter"]}>
```

Comment:
What's the plan for filling these in? Do they leave empty lines in the docs page?

Reply:
not in this section but I did notice the filter mid-unordered list creates a break. fixing...

Reply:
updated